### PR TITLE
`init-cairo1` docs

### LINKED ISSUE #1584: Add documentation for `init-cairo1`
Create a new section "Cario 1 support" if it doesn't exist already

### PR DESCRIPTION
**Pull request by Arcticae**
closes #1584

### CODE REVIEW

File: website/docs/tutorials/08-cairo-1-support/01-project-initialization.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+This template contains:
+
+- `src` directory
+    - `cairo_project.toml` which is needed for compilation 
+    - `main.cairo` file with one function definition
+    - `lib.cairo` file which defines the module
+- `test` directory
+    - Single test file with one test for function defined in `main.cairo`
+- `protostar.toml` containing information about the project
```
Comment (piotmag769) [inline website/docs/tutorials/08-cairo-1-support/01-project-initialization.md:31]:
```suggestion
- `src` – a directory for the src module
    - `main.cairo` – a module source file
    - `lib.cairo` – a module definition file
    - `cairo_project.toml` – a module configuration file
- `tests` – a directory for tests
    - `test_main.cairo` – a test file for the src module (tests the code from `main.cairo`)
- `protostar.toml` – a project configuration file
```

Comment (piotmag769) [reply website/docs/tutorials/08-cairo-1-support/01-project-initialization.md:31]: We can also consider adding description for users on how to create modules.

Maybe renaming src directory to e.g. `template_module` will give be a good idea – the name `src` suggests that 
all of the source code should be put in this folder. 

We could also put it one level deeper – `src/template_module/`. That way users won't get lost and will just follow the pattern to add another modules into the `src` folder.

File: website/docs/tutorials/08-cairo-1-support/01-project-initialization.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## The protostar.toml
+Apart from the usual things you find in `protostar.toml`, there is `linked-libarires` entry which is used to find cairo 1 modules in tests and building.
```
Comment (piotmag769) [inline website/docs/tutorials/08-cairo-1-support/01-project-initialization.md:34]:
```suggestion
Apart from the usual things you can find in `protostar.toml`, there is a `linked-libarires` entry which is used to find cairo 1 modules in tests and building.
```

File: website/docs/tutorials/08-cairo-1-support/01-project-initialization.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+This makes it possible for you to include other modules from your dependencies, if they are correct cairo 1 modules (with their own module definition and `cairo_project.toml`).
```
Comment (piotmag769) [inline website/docs/tutorials/08-cairo-1-support/01-project-initialization.md:35]: Not sure about this comma, worth asking someone knowledgeable about this.
```suggestion
This makes it possible for you to include other modules from your dependencies if they are correct cairo 1 modules (with their own module definition and `cairo_project.toml`).
```

Comment (karol-bisztyga) [reply website/docs/tutorials/08-cairo-1-support/01-project-initialization.md:35]: should be removed

File: website/docs/tutorials/08-cairo-1-support/01-project-initialization.md (hunk 1 above)
Comment (karol-bisztyga) [reply website/docs/tutorials/08-cairo-1-support/01-project-initialization.md:31]: Well, this doc is about project **initialization**, so playing with modules feels a bit out of scope here. I think we should approach this in a more specific (not generic) way and just tell people how to set up a quick project so they can write code.

**Review by karol-bisztyga — APPROVED** [review]

File: website/docs/tutorials/08-cairo-1-support/01-project-initialization.md (hunk 1 above)
Comment (piotmag769) [reply website/docs/tutorials/08-cairo-1-support/01-project-initialization.md:31]: Okay, agree with that.

Comment (Arcticae) [reply website/docs/tutorials/08-cairo-1-support/01-project-initialization.md:31]: I think we could discuss creating a doc for how to create a cairo1 module. Although i would much rather peek at the cairo1 docs first to see their approach. We could schedule that for the next milestone.

Comment (Arcticae) [reply website/docs/tutorials/08-cairo-1-support/01-project-initialization.md:31]: #1591

**Review by piotmag769 — APPROVED** [review]

**Pull request merged**